doc/sphinx/proof-engine/ssreflect-proof-language.rst

```diff
@@ -114,14 +114,14 @@ compatible with the rest of |Coq|, up to a few discrepancies:
 + New keywords (``is``) might clash with variable, constant, tactic or
   tactical names, or with quasi-keywords in tactic or vernacular
   notations.
-+ New tactic(al)s names (``last``, ``done``, ``have``, ``suffices``,
-  ``suff``, ``without loss``, ``wlog``, ``congr``, ``unlock``)
++ New tactic(al)s names (:tacn:`last`, :tacn:`done`, :tacn:`have`, :tacn:`suffices`,
+  :tacn:`suff`, :tacn:`without loss`, :tacn:`wlog`, :tacn:`congr`, :tacn:`unlock`)
   might clash with user tactic names.
 + Identifiers with both leading and trailing ``_``, such as ``_x_``, are
   reserved by |SSR| and cannot appear in scripts.
-+ The extensions to the rewrite tactic are partly incompatible with those
++ The extensions to the :tacn:`rewrite` tactic are partly incompatible with those
```

Comment:
Here you changed your strategy w.r.t. `rewrite`. Writing `(ssreflect)` in the text was ugly, so maybe we are missing some infrastructure to distinguish the text and the link target.

Reply:
Oh, I understand now that it was more or less the Coq `rewrite` that was meant in this paragraph. So maybe let's leave things as they are.

Reply:
Doesn't it work to include the target in brackets, `` like :tacn:`this <target-name>` ``?

Reply:
Indeed, here it was meant to be the `rewrite` from the Tactics chapter. I agree that having `(ssreflect)` in the text is a bit ugly, but OTOH I would really like to see an effort to unify at least the tactics that share the same name. Everybody seems to think that SSR's rewrite is better so what would be missing to make it the default rewrite. The syntax discrepancy could be solved using a compatibility option, the selection algorithm as well. Are there some features from `rewrite` that wouldn't be supported by `rewrite (ssreflect)`?

Reply:
That being said, Clément's trick above does work so let me apply it in a few places.